<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FReservationsCommon</class>
 <widget class="QWidget" name="FReservationsCommon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="10">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Search by</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbArrival">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>Arrival</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbDeparture">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>Departure</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Arrival starting</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Arrangement</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Room</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="EQLineEdit" name="leState">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
       <property name="EnableHiddenText" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>State</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="EQLineEdit" name="leRoom">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
       <property name="EnableHiddenText" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ending</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Cardex</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="EQLineEdit" name="leStatus">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
       <property name="EnableHiddenText" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="6">
      <widget class="EQLineEdit" name="leCardex">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
       <property name="EnableHiddenText" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="EDateEdit" name="deStart"/>
     </item>
     <item row="0" column="3">
      <widget class="EDateEdit" name="deEnd"/>
     </item>
     <item row="0" column="9">
      <widget class="EQLineEdit" name="leGroupNum">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="EnableHiddenText" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QCheckBox" name="chOnlyWithCardex">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>With cardex only</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="EQLineEdit" name="leArrangement">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
       <property name="EnableHiddenText" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_9">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Group #</string>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QCheckBox" name="chOnlyGroup">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Groups only</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Booking</string>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="EQLineEdit" name="leBooking">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
